BUILD.md

```diff
@@ -0,0 +1,88 @@
+# Building DLAF
+
+## Build using Spack
+
+Example installation:
+
+`spack install dla-future ^[virtuals=blas,lapack] intel-oneapi-mkl`
+
+or you can also build C API compatible with ScaLAPACK with:
+
+`spack install dla-future +scalapack ^[virtuals=blas,lapack,scalapack] intel-oneapi-mkl`
+
+You can go even further with a more detailed spec like this one, which builds dla-future in debug mode, using the clang compiler, specifying that the pika on which it depends has to be built
+in debug mode too, and that we want to use MPICH as MPI implementation, without fortran support (because clang does not support it).
+
+`spack install dla-future %clang build_type=Debug ^pika build_type=Debug ^mpich ~fortran`
+
```

Comment:
Do you think it makes sense to add a link to the spack docs (https://spack.readthedocs.io/en/latest/) and say "see spack docs for more information"?

Reply:
c2a5fb003f7b9ba5b34664dc090fffcd8a0794b1